apps/meteor/app/federation/server/lib/http.js

```diff
@@ -19,11 +19,17 @@ export async function federationRequest(method, url, body, headers, peerKey = nu
 
 	httpLogger.debug(`[${method}] ${url}`);
 
-	return MeteorHTTP.call(method, url, {
-		data,
-		timeout: 2000,
-		headers: { ...headers, 'x-federation-domain': getFederationDomain() },
-	});
+	try {
```

Comment:
Is this try/catch necessary since we are just rethrowing the error?

Reply:
Nah, not really necessary